frontend/src/components/shared/ResourceCard.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import './ResourceCard.css';
 
-const ResourceCard = ({ image, title, description, link, bg }) => {
+const ResourceCard = ({ image, title, description, link, bg, buttonStyle, buttonText }) => {
   const cardClass = bg ? 'card-with-bg' : 'card-without-bg';
 
   return (
     <div className={`resource-card ${cardClass}`}>
       <img src={image} alt={title} />
       <h2>{title}</h2>
       <p>{description}</p>
-      <a href={link} className="learn-more-link">Learn More</a>
+      <button onClick={() => window.location.href = link} className={`learn-more-button ${buttonStyle}`}>
```

Comment:
<img width="537" alt="Screenshot 2023-10-19 at 2 29 23 PM" src="https://github.com/arzucaner/when-women-understand/assets/84030413/ccb5f7a1-3a1c-4e86-aa11-b2bca36c51d6">
For some reason, the `learn-more-button` is throwing undefined and appears like this in the browser